pages/implementation/reference/swift.mdx

```diff
@@ -74,6 +74,26 @@ avo.signupStart(referral: "direct")
 
 Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the <Link passHref href="/implementation/inspector/sdk/ios"><a>Avo Inspector</a></Link> to verify that your implementation is correct. If you don't want to use Avo Inspector you can use the <a href="https://github.com/avohq/ios-analytics-debugger">standalone visual debugger</a>.
 
+#### Step 5. Suppress warnings in the Avo files
+
+Because of the nature of the code generation Avo files can sometimes contain warnings, that do not affect the execution of the code, but may distract you from the product development.
```

Comment:
```suggestion
Because of the nature of the code generation Avo files can sometimes contain warnings, that do not affect the execution of the code, but may distract you.
```